From 1ec2848c205249420d64d7924f2ee1840efff140 Mon Sep 17 00:00:00 2001
From: Jeremy Stanley <fungi@yuggoth.org>
Date: Sat, 16 Sep 2017 15:47:03 +0000
Subject: [PATCH] Add FAQ entry about --headers values

Be as clear as possible about what the --headers command-line option
or headers configuration option does, and what sorts of values are
valid for it.
---
 FAQ | 24 +++++++++++++++++++++++-
 1 file changed, 23 insertions(+), 1 deletion(-)

diff --git a/FAQ b/FAQ
index 1e0ff03..280c0af 100644
--- FAQ
+++ FAQ
@@ -2,7 +2,7 @@
  Frequently Asked Questions About the Weather Utility
 ======================================================
 
-:Copyright: (c) 2006-2012 Jeremy Stanley <fungi@yuggoth.org>. Permission
+:Copyright: (c) 2006-2017 Jeremy Stanley <fungi@yuggoth.org>. Permission
             to use, copy, modify, and distribute this software is
             granted under terms provided in the LICENSE file distributed
             with this software.
@@ -54,3 +54,25 @@ As of the 2.0 release, this question is no longer relevant.
 ---------------------------------------------------------------
 As of the 2.0 release, this is no longer necessary. See FAQ entries #2
 and #3 for more detail.
+
+7. What values are valid for a --headers list?
+----------------------------------------------
+The default set it uses if you don't override it yourself on the command
+line or in configuration is as follows::
+
+  heat_index
+  precipitation_last_hour
+  relative_humidity
+  sky_conditions
+  temperature
+  weather
+  wind
+  windchill
+
+These are a case-insensitive match against the start of lines in a
+decoded METAR up to the first colon (:) with underscores (_) replaced by
+spaces. You can see the full METAR for a given condition report by
+passing --verbose or by observing one directly (perhaps by looking in
+your *datacache* directory). Unfortunately I haven't found any proper
+specification for the decoded METAR format used by the NWS so know of no
+comprehensive list of what lines might appear.
